packages/morph/src/styles/editor.scss

```diff
@@ -170,100 +168,106 @@ html {
+  aside {
+    border: none !important;
+  }
+
+  background-color: #fff;
+  position: absolute;
+  top: 0;
+  right: -40rem; 
+  width: 31rem;
+  height: calc(100vh - 2.3rem);
+  padding: 1rem 1rem; 
+  box-sizing: border-box;
+  transition: right 0.3s ease-in-out;
+  overflow-y: auto;
   display: flex;
   flex-direction: column;
-  padding-bottom: 1rem;
 
-  .editor-content {
+  h2 {
+    margin-bottom: 1rem;
+    font-family: $font-heading;
+  }
+
+  .notes-columns {
     display: flex;
+    gap: 0.5rem;
     flex: 1;
-    position: relative;
-    min-height: 0;
-    overflow: hidden;
-    
-    .editor {
-      flex: 0 0 100%;
-      transition: flex-basis 0.3s ease;
+    padding-right: 0.25rem;
+    overflow-y: auto;
+    overflow-x: auto;
+
+    &::-webkit-scrollbar {
+      width: 6px;
+    }
+    &::-webkit-scrollbar-track {
+      background: #f1f1f1;
+      border-radius: 3px;
+    }
+    &::-webkit-scrollbar-thumb {
+      background: $gray-medium;
+      border-radius: 3px;
+      &:hover {
+        background: #bbbbbb;
+      }
     }
 
-    .notes-panel {
-      flex: 0 0 25%; 
-      transform: none !important;
-      transition: transform 0.3s ease;
-      padding: 1rem;
-      border: 1px solid $gray-light;
-      border-radius: 8px;
+    .notes-column {
+      flex: 1;
       display: flex;
       flex-direction: column;
-      height: calc(100vh - 4.8rem);
-      min-height: 400px;
-      position: sticky;
-      top: 1rem;
-      overflow-x: auto;
-      
-      
-      h2 {
-        margin-bottom: 1rem;
-        flex-shrink: 0;
-        font-family: $font-heading;
-      }
+      gap: 0.5rem;
+    }
+  }
 
-      .notes-columns {
-        display: flex;
-        gap: 1rem;
-        overflow-y: auto;
-        overflow-x: auto;
-        flex: 1;
-        padding-right: 0.5rem;
-        
-        &::-webkit-scrollbar {
-          width: 6px;
-        }
-
-        &::-webkit-scrollbar-track {
-          background: #f1f1f1;
-          border-radius: 3px;
-        }
-
-        &::-webkit-scrollbar-thumb {
-          background: $gray-medium;
-          border-radius: 3px;
-
-          &:hover {
-            background: #bbbbbb;
-          }
-        }
-
-        .notes-column {
-          flex: 1;
-          display: flex;
-          flex-direction: column;
-          gap: 1rem;
-        }
-      }
+  .note-item {
+    background-color: #ffffff;
+    border-radius: 0.5rem;
+    padding: 1rem;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+    font-family: $font-body;
+    font-weight: $normalWeight;
+  }
+}
 
-      .note-item {
-        background-color: #ffffff;
-        border-radius: 0.5rem;
-        padding: 1rem;
-        box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
-        transform: translateZ(0);
-        transition: transform 0.2s ease, font-weight 0.2s ease;
-        font-family: $font-body;
-        font-weight: $normalWeight;
-      }
-    }
+.workspace-container.show-notes {
+  grid-template-columns: 77.5rem 31.5rem;
```

Comment:
ideally we should use `1fr` or `auto`

Bounding `$mobile` are usually 800px, and `$tablet` is from `800-1400px`, and desktop is 1400px and above

downstream cc @aarnphm